src/Scrutor/ServiceCollectionExtensions.Scanning.cs

```diff
@@ -32,6 +32,25 @@ public static IServiceCollection Scan(this IServiceCollection services, Action<I
             return services.Populate(selector, RegistrationStrategy.Append);
         }
 
+        public static IServiceCollection Scan(this IServiceCollection services, Action<ITypeSelector> action)
```

Comment:
How does this overload work? When will the action argument be `ITypeSelector` and when is it `IAssemblySelector`?